WikiWatershed/model-my-watershed
# Update some front-end libraries for security

## Overview

The following libraries were upgraded:

* bootstrap-select
* mocha
* node-sass
* testem

To fix security issues in:

* socket.io
* node-notifier
* ini
* yargs-parser
* http-proxy
* node-sass
* bootstrap-select

### Notes

Security issues with `src/tiler/package.json` were not addressed since they are only used during build-time.

## Testing Instructions

* Check out this branch and run `./scripts/yarn.sh && ./scripts/bundle.sh --vendor`
* Go to http://localhost:8000/ and select a shape, ensure analysis works
